content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,201 @@
+* `rdtoolkit_state` | Text
+     * The RDT status.
+     * Auto filled by the CHT integration.
+* `rdtoolkit_time_started` | Text
+     * The date and time when the RDT started.
+     * Auto filled by the CHT integration.
+* `rdtoolkit_time_resolved` | Text
+     * The date and time when the RDT result is ready to capture.
+     * Auto filled by the CHT integration.
+
+### Sample form for provisioning RDT
+![Provisioning RDT form](rdtoolkit-povision-form.png)
+
+### Capturing RDT result form
+It is required to add the following labels in the “appearance” tag:
+* `rdtoolkit-capture`: To attach the Enketo widget that connects with the RDToolkit app.
+* `rdtoolkit-action-btn`: To attach the action button that captures the RDT result, this button will open the RDToolkit.
+
+This form has the following mandatory fields:
+* `patient_uuid` | Text
+    * Patient's unique ID in the system.
+    * Auto filled when using the contact lookup field.
+* `patient_id` | Text
+    * Patient's representative ID, for example, can be the ID in the health facility's records.
+    * Auto filled when using the contact lookup field.
+* `patient_name` | Text
+    * Patient's first name and / or last name
+    * Auto filled when using the contact lookup field.
+* `rdtoolkit_session_id` | Text
+    * The unique ID for the RDT.
+    * Auto filled by CHT integration.
+* `rdtoolkit_results` | Text
+    * JSON representation of the RDT result
+    * Auto filled by CHT integration.
+* `rdtoolkit_results_description` | Text
+    * Translation of the results
+    * Auto filled by CHT integration.
+* `rdtoolkit_image` | Text
+    * Base64 image encoding.
+    * Auto filled by CHT integration and will render the image.
+    * **appearance**: hidden
+    * **instance::type**: binary
+
+Optional fields:
+* `rdtoolkit_state` | Text
+    * The RDT status.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_started` | Text
+    * The date and time when RDT started.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_resolved` | Text
+    * The date and time when the RDT result is ready to capture.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_read` | Text
+    * The date and time when RDT result was captured.
+    * Auto filled by CHT integration.
+
+### Sample form for capturing RDT results
+![Capturing RDT result form](rdtoolkit-capture-form.png)
+
+## Create a task
+A [task](https://docs.communityhealthtoolkit.org/apps/features/tasks/) is added when a new provisioning RDT form is created, and it will be resolved when a capture RDT result form is saved. 
+The task needs to copy the `rdtoolkit_session_id` from the provision form to the capture form in order to retrieve RDT results from the RDToolkit app. 
+
+The following is an example of a task implementation:
+
+```javascript
+{
+  name: 'rdtoolkit-capture-results',
+  icon: 'icon-follow-up',
+  title: 'task.rdtoolkit.capture.title',
+  appliesTo: 'reports',
+  appliesToType: ['rdtoolkit-provision'], // form
```

Comment:
This `// form` comment looks like it could use some elaboration.

Reply:
// The form associated with the task creation.